core/templates/dev/head/css/oppia.css

```diff
@@ -2150,10 +2298,74 @@ oppia-parameter {
   word-wrap: break-word;
 }
 
+.modal-content,.modal-dialog {
```

Comment:
Perhaps there should be a comment above this section to say these are bootstrap overrides for modals? Ditto similarly elsewhere.

Reply:
Added comment lines and segmented it Thanks!